python/lsst/verify/bin/jobReporter.py

```diff
@@ -50,7 +49,7 @@ def main(repository, collection, metrics_package, spec, dataset_name):
     if len(jobs) == 0:
         raise RuntimeError('Job reporter returned no jobs.')
     for k, v in jobs.items():
-        filename = f"{metrics_package or 'all'}_{spec}_{k}_{time.time()}.json"
+        filename = f"{metrics_package or 'all'}_{spec}_{k}.json"
```

Comment:
Actually, I have a separate request: if you're changing the filename format anyway, would you be willing to make the extension `.verify.json`? That would make it easier to look for these files in lsst-dm/jenkins-dm-jobs#760.

Reply:
Done